src/pykiso/logging_initializer.py

```diff
@@ -151,10 +137,10 @@ def initialize_logging(
     # for all report types add a StreamHandler
     stream_handler = logging.StreamHandler(stream)
     stream_handler.setFormatter(log_format)
-    stream_handler.setLevel(LEVELS[log_level])
-    if not verbose:
-        # filter internal log levels
-        stream_handler.addFilter(InternalLogsFilter())
+    # disable internal logs if no verbose is wanted
+    stream_handler.setLevel(
+        LEVELS[log_level] if not verbose else (LEVELS[log_level] - 1)
+    )
     root_logger.addHandler(stream_handler)
 
     root_logger.setLevel(LEVELS[log_level])
```

Comment:
```suggestion
    root_logger.setLevel(LEVELS[log_level] - 1 if (verbose or log_path) else LEVELS[log_level] )
```
Else the log level will never be at internal level.

Reply:
Done ;)